main.go

```diff
@@ -81,6 +81,14 @@ const (
 	// Server shutdown grace period.
 	gracePeriod = 2 * time.Minute
 )
+const appName = "observatorium-api"
```

Comment:
We should use the `--debug.name` instead of the constant because this repo is also whitelabeled as `lokistack-gateway` in Loki Operator managed deployments.

Reply:
oh! good to know that. will do the change.